tests/components/composer.test.ts

```diff
@@ -1206,6 +1207,26 @@ describe("composer", () => {
     expect(fixture.querySelectorAll(".o-grid div.o-composer")).toHaveLength(0);
   });
 
+  test("The composer should be closed before selecting headers", async () => {
+    composerEl = await typeInComposerGrid("=");
+    composerEl.dispatchEvent(new MouseEvent("click"));
```

Comment:
why is this necessary ? Don't you have the focus once you start typing?

Reply:
Because I type in "=" I need to click the cell again to make it into editing mode (otherwise it is selecting mode). I've changed into "Hello" and this click event is deleted now